joomla/joomla-cms
# Fix the backend logout button issue

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/37109[](https://user-images.githubusercontent.com/82047120/156143888-5484a57c-8d46-40f0-8dd4-0f70a9d781fa.png)[](https://user-images.githubusercontent.com/82047120/156147977-a8c9326c-6127-4182-adfb-2d8887f7ac46.jpeg)

Summary of Changes
Put the logout button outside the dropdown menu.

Testing Instructions
After applying the patch, see the header of home page , you

Actual result BEFORE applying this Pull Request
![68747470733a2f2f6973737565732e6a6f6f6d6c612e6f72672f75706c6f6164732f312f39363334316334343432313938626239366230323562663934356465363136322e706e67](https://user-images.githubusercontent.com/82047120/156228472-0d13f347-fa82-4908-9225-b40b19f89719.png)





Expected result AFTER applying this Pull Request
![Web capture_1-3-2022_153056_localhost](https://user-images.githubusercontent.com/82047120/156228265-98417059-0563-4b39-80e4-6c439969faae.jpeg)


Documentation Changes Required
No changes required
